### ISSUE TITLE
Add final evaluation scripts for Total Segmentator

### ISSUE THREAD

**Issue by surajpaib** [project-lighter/CT-FM#10]
Final evaluation will include a training script for TS with different configurations:

- [ ] Decoder-only training (V2)
- [ ] Few-shot training (1%, 2%, etc.) (V2)
- [ ] Grouped classes (organs, muscles, ribs, etc. ) training (V1)
- [ ] Merlin comparison (V2)
- [ ] Full-finetune (V2) 
- [ ] Training time-taken for 90 dice?
- [ ] Pre-training ablations (V2)
- [ ] SSL pre-training influence over time (V2)

Baselines to compare against:

 **TSv1**
- [ ] SuPreM finetuned
- [ ] SAT-Pro-Ft/SAT-Pro/SAT Nano [Group comparison + Change the test set]
- [ ] Random initialization where applicable

**TSv2 grouped classes** 
- [ ] Merlin
- [ ] SuPreM finetuned?
